x-pack/plugins/log_explorer/public/components/flyout_detail/sub_components/highlight_field_description.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiIconTip } from '@elastic/eui';
+import { EcsFlat } from '@kbn/ecs';
+import { FieldIcon } from '@kbn/react-field';
+import React from 'react';
+
+export function HighlightFieldDescription({ fieldName }: { fieldName: string }) {
+  const { short, type } = EcsFlat[fieldName as keyof typeof EcsFlat];
+
+  if (!short) return null;
+
+  const title = (
+    <EuiFlexGroup alignItems="center" gutterSize="s">
+      {type && (
```

Comment:
Should we have here a default type?

Reply:
I would say no, because it could be misleading about the type of field if we don't have it. I'd rather hide the icon and leave the field name alone instead of showing a wrong field type based on a default value, wdyt?

Reply:
I expressed myself wrongly, most accurate a default icon for the type, something like `unknown` 😅 (don't know if that exists or not). Something like:
<img width="482" alt="image" src="https://github.com/elastic/kibana/assets/1313018/22bab78b-eb33-4fd6-bcd2-e8b835bc00f5">

More of a visual nitpick, if we have a default icon for the fields all of them would be kind of aligned whilst if we just hide the icon will be like something is missing.

Reply:
Your first comment was clear 👍 
I still believe we should not show an Unknown icon for a matter of familiarity.
All the field icons come from [these tokens](https://eui.elastic.co/#/display/icons#tokens), which represent the type of fields. If we show an icon not part of that set associated with a field, it might be confusing for the user.

In terms of alignment, this is standalone content, the user won't see simultaneously multiple tooltips, so there is no alignment issue on this design.

Also, the component already set a default icon in case the passed field (most likely a custom one) has not a corresponding type (which is unlikely), so this check is more of a gate for those edge cases we could incur when we'll update the logic to get the description and type.